pkg/migrations/op_drop_table.go

```diff
@@ -25,6 +25,7 @@ func (o *OpDropTable) Complete(ctx context.Context, conn db.DB, tr SQLTransforme
 }
 
 func (o *OpDropTable) Rollback(ctx context.Context, conn db.DB, tr SQLTransformer, s *schema.Schema) error {
+	s.UnRemoveTable(o.Name)
```

Comment:
worth adding a comment on why this is done?

Reply:
comment added